app/pubspec.yaml

```diff
@@ -21,7 +21,9 @@ dependencies:
   device_info_plus: ^11.2.0
   flip_card: ^0.7.0
   flutter:
+  
```

Comment:
The space should be where it was before, in the end of `flutter:`, think of it as a code in python.

So actually it is after the `sdk: flutter` where the `flutter:` is finished